apirest.php

```diff
@@ -46,11 +46,8 @@
  */
 global $GLPI, $GLPI_CACHE;
 
-define('GLPI_ROOT', __DIR__);
 ini_set('session.use_cookies', 0);
 
-include_once(GLPI_ROOT . "/inc/based_config.php");
-
 // Init loggers
```

Comment:
Initialization of the logger, the error handler and the cache service are probably done by the legacy configurators now. The corresponding code could be removed.